sharepoint_client.py

```diff
@@ -0,0 +1,53 @@
+import requests
+import time
+from requests_ntlm import HttpNtlmAuth
+from urllib.parse import urljoin
+from configuration import Configuration
+from sharepoint_utils import print_and_log
+
+domain =   "[REDACTED]"
```

Comment:
not the usual Pythonic style, you can look at `black` for automatically fix your style

Reply:
done